src/query/config/src/inner.rs

```diff
@@ -145,6 +149,9 @@ pub struct QueryConfig {
     pub tenant_id: String,
     /// ID for construct the cluster.
     pub cluster_id: String,
+    // ID for the query node.
+    // This only initialized when InnerConfig.load().
```

Comment:
```suggestion
    // This only initialized when InnerConfig::load().
```

Reply:
Thanks.
The CI already running, you can help me fix it in your PR 😄

Reply:
Wait the unit test passed, will merge into main.